packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -94,6 +96,16 @@
  */
 - (UIViewController *)createRootViewController;
 
+/**
+ * It assigns the rootView to the rootViewController
+ * By default, it assigns the rootView to the view property of the rootViewController
+ *
```

Comment:
I would add slightly more explanation on why this is needed. 
Just something along the lines of:
```
If you are not using a simple UIViewController, then there could be other methods to use to setup the rootView. For example ....
```